### ISSUE TITLE
ADFS/SSO Fix Blank Screen

### ISSUE THREAD

**Issue by crollcomp** [ramboxapp/community-edition#2582]
I would like to report that I was able to find a "what worked for me" solution to the ADFS/SSO issue that has plagued many users.  My org had a unique situation where we have an internal ADFS server, and an external ADFS, since we are an Office365 shop.  Your results may vary.

First off, allow me to say, you should have some technical background and understand what you are changing here.

I added an entry to my hosts file, listing the internal IP address of my ADFS server, and the host being the DNS name of my ADFS server.

Windows hosts file is located C:\Windows\system32\drivers\etc\hosts
Mac hostfile: terminal `sudo nano /etc/hosts`

**Comment by TheKangaroo** [comment]
Unfortunately, this does not work for me.
I think the problem is whether your ADFS server points you to a page to enter your credentials or opens a popup box (I think it's HTTP Authentication) to enter your credentials.
I think you see the blank page if you was asked for your credentials via popup box.

**Comment by MrJohanssons** [comment]
I don't get it, I'm only able to logon to Teams when not connected to the VPN. Does that mean I should add a manual entry in my hosts file that points to the external ADFS IP?

**Comment by crollcomp** [comment]
> I don't get it, I'm only able to logon to Teams when not connected to the VPN. Does that mean I should add a manual entry in my hosts file that points to the external ADFS IP?

Yeah that's correct. If you go back to your office or connect to a vpn and have issues, then you'd want to comment out your host file entry.

**Comment by tobmaster** [comment]
I can confirm that this was the issue for me too. In the office we have an adfs server that takes care of the teams authentication. When I disconnected from the company wifi and used tethering over mobile network I could login without any problem. So there is no dialog or something like that.
In the browser this works even without the trick. So there is still something that rambox is doing differently.
